pep-9999.rst

```diff
@@ -0,0 +1,520 @@
+*new* production-ready release stream that is more specifically tailored to the
+interests of operating environments that are in a position to consume new
+releases as fast as the CPython core team is prepared to produce them.
+
+
+Why "continuous beta" rather than "continuous alpha"?
+-----------------------------------------------------
+
+The code quality standards upheld by the CPython code review process and
+BuildBot fleet make the "beta" label more suitable than the "alpha" label.
+
+The "production ready, but subject to change" caveat also aligns well with the
+"beta" term.
+
+
+Why not use Calendar Versioning for the rolling release stream?
+---------------------------------------------------------------
+
+Steve Dower's initial write-up of this proposal [1_] suggested the use of
+calendar versioning for the rolling release stream (so Python 2019.12 rather
+than 3.9.0b1, etc).
+
+Paul Moore pointed out [2_] two major practical problems with that proposal:
+
+* it makes it hard for users of the calendar-based versions to know where they
+  stand in relation to the traditionally numbered versions
+* it breaks ``Python-Requires`` metadata processing (since all calendar
+  versions would appear as newer than any standard version)
+
+This PEP aims to address both of those problems by using the established beta
+version numbers for the rolling releases.
+
+The beta numbering approach also avoids other questions raised by the calendar
+versioning concept, such as how ``sys.version_info``, ``Py_VERSION_HEX``,
+``site-packages`` directory naming, and installed binary naming would work.
+
+
+Implications for CPython core development
+-----------------------------------------
+
+The major change for CPython core development is the need to keep the master
+branch more consistently release ready.
+
+While the main requirement for that would be to keep the stable BuildBot fleet
+green, there would also be encouragement to keep the development version of
+the documentation up to date for the benefit of users of the rolling beta
+releases.
+
+
+Implications for Python library development
+-------------------------------------------
+
+By using the "beta" labelling rather than the incremental feature release
+numbering proposed in PEP 598, the hope would be that consumers of these
+new rolling releases would realise that they're likely going to need to build
+their own wheel archives from source, and will generally be more prone to
+encountering library compatibility issues when updating to a new release.
+
+Library authors who actually want to support the beta stream would have the
+option of testing against the latest beta release in their pre-merge test
+matrices (just as they test against the latest maintenance release of previously
+published versions), with the CPython nightly builds offered by some CI
+providers used solely in an advisory capacity for early detection of potential
+compatibility problems.
```

Comment:
We can also add that official releases make it easier for CI providers to adopt our latest code (because they can trust that we've done at least basic validation) compared to build-from-master.

Reply:
Mentioned